lib/mix/test/mix/tasks/run_test.exs

```diff
@@ -11,17 +11,6 @@ defmodule Mix.Tasks.RunTest do
     Mix.Project.push(MixTest.Case.Sample)
   end
 
-  test "loads configuration", context do
```

Comment:
What do you think about keeping this test and add a `# TODO: Remove on v2.0.` instead?

Reply:
Done.